app/components/windows/go-live/GoLiveChecklist.tsx

```diff
@@ -58,7 +58,11 @@ export default class GoLiveChecklist extends TsxComponent<Props> {
 
   private getHeaderText() {
     if (this.error) {
-      return $t('Something went wrong');
+      if (this.view.info.checklist.startVideoTransmission === 'done') {
+        return $t('You stream has been started with issues');
```

Comment:
this should probably be something more clear like `Your stream has started, but there were issues with other actions taken`

Reply:
^ Yeah Sean's suggestion is good.  Let's use that copy.